### DISCUSSION TITLE
Bar chart: Scrollable charts (Y-axis) for large data sets with sticky X-axis

### DISCUSSION THREAD

**Discussion by Digvijayrao-KF** [apexcharts/apexcharts.js#4345] [General]
Hello, **@junedchhipa** and other esteemed code owners! 🌟

I appreciate your attention to detail and your commitment to enhancing **ApexCharts**. The feature request for vertical scrolling on the y-axis for large data sets (with more than 10+ labels) while keeping the x-axis sticky for bar charts is indeed a valuable addition.

The issue you've referenced ([#2853](https://github.com/apexcharts/apexcharts.js/issues/2853), https://github.com/apexcharts/apexcharts.js/issues/4202) highlights this need, and I understand how crucial it is for end-users. When dealing with a defined width and height of the wrapper container, having a scrollbar feature for better readability and user experience can significantly enhance the chart's usability.

While I don't have direct insight into ApexCharts' development roadmap, I recommend keeping an eye on the issue thread for updates. The ApexCharts team is responsive, and they may provide information on whether this feature is planned and an estimated timeline.

In the meantime, let's hope that this enhancement finds its way into the next release! 🚀

Also, if this requirement is manageable in **apexcharts** at the moment, please highlight the same accordingly. 

CC @rosco54 , @danielchalmers

Thanks.

**Comment by danielchalmers** [comment]
is this ChatGPT?

**Reply by Digvijayrao-KF** [reply]
Hello, I am a human with a genuine interest in this requirement. Our organization currently utilizes this package for it's esteemed products. I am curious to know if this feature request will be taken into consideration for further optimization of **apexcharts**.

**Comment by yogeshvats0001** [comment]
Any updates on addition of scrollbar in yAxis and making the xAxis sticky to more the data more readable and better user experience. We're struggling with apexcharts library in the project. Please include this in future enhancements.

**Comment by jschank** [comment]
I, too, am interested in this. I have a timeline with range bars, but some instance of my chart have a large number of y-axis values. I would like to make that axis scrollable, while maintaining visibility of the x-axis at all times.

**Reply by yogeshvats0001** [reply]
Hi @jschank 
I have tried several ways but nothing worked for me. So, I used highcharts library for fixed x-axis.
